docker/server/Dockerfile

```diff
@@ -0,0 +1,28 @@
+#set up python runtime environment
+FROM python:3.6.2-slim
```

Comment:
What does slim do?

Reply:
@andela-sjames The slim image is for when you want the minimal packages to run python. I was having space constraints so I decided to use it. But the default tag is more recommendable for production.